### ISSUE TITLE
Save previewed files in C. as message drafts

### ISSUE THREAD

**Issue by alexara** [siacs/Conversations#3239]
#### General information

* **Version:** 2.3.2
* **Conversations source:** PlayStore

#### Steps to reproduce

1. Click on the clip
2. Take pictures
3. The pictures are now shown as previews in the input field
4. Leave the chat, i.e. go to the Conversations overview *
5. Go back to the chat

* by accident; in order to reply in another chat; etc.

#### Expected result

The previews should have been saved as a message draft.
That would also be consistent with text messages, which are always saved.

#### Actual result

The previews have vanished. The taken pictures are lost.

**Comment by Echolon** [comment]
Yes, this is really annoying for audios. I frequently loose one or two minute long audios because I just accidentially click on the screen. So being able to continue the already recorded audio would be awesome!

One possible setup is described here from our beloved WhatsApp: https://faq.whatsapp.com/en/android/23684991/?category=5245251 and you can see it here for one second: https://youtu.be/CSHrAcEYqpo?t=412 _(It's the lock symbol which pops up when holding the reconding button)_
